spec/views/coordinators/show.html.erb_spec.rb

```
@@ -0,0 +1,56 @@
+# spec/views/products/show.html.erb_spec.rb
+
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe 'coordinators/show.html.erb', type: :view do
+  describe 'populate page and links' do
+    before do
+      @coordinator = Coordinator.create(name: 'Joe Lally',
+                                        email: 'email@email.com')
+      @tutors = [Tutor.create(first_name: 'Bob', last_name: 'Smith'),
+                 Tutor.create(first_name: 'Tom', last_name: 'Brady')]
+      @students = [Student.create(first_name: 'Mike', last_name: 'White'),
+                   Student.create(first_name: 'Alex', last_name: 'Fallah')]
```

As we add more required fields, it will become far too cumbersome to use `Student.create` and `Tutor.create`, and we will 100% want to pivot to using FactoryGirl.

---

Did some digging after rebasing on my most recent commit. They'll be built and saved to the variables just fine, but when rspec tries to persist them to the db it will silently fail.

This is valuable though. We shouldn't create or persist anything more than we need to, so knowing that we can *build* objects without persisting them and still have this work is 💯 . We should be doing that.

So we can use `.new` instead. Or we can use `build(:student)` to get FactoryGirl to build one for us but not persist it, iirc.

---

If we use `.new` we need to give it an ID.

This is all not a huge deal, though. Do whatever works!